inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/hive/pom.xml

```diff
@@ -31,8 +31,6 @@
 
     <properties>
         <inlong.root.dir>${project.parent.parent.parent.parent.parent.basedir}</inlong.root.dir>
-        <hive3x.version>3.1.3</hive3x.version>
-        <hive2x.version>2.2.0</hive2x.version>
```

Comment:
You can't delete it. This connector will release hive2 and hive3 version by use different mvn profile.